openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -2590,7 +2590,7 @@ def _assert_time_zone_is_valid(self, time_zone_info):
         self.assertIn(time_zone_name, common_timezones_set)
         self.assertEqual(time_zone_info['description'], get_display_time_zone(time_zone_name))
 
-    @ddt.data((ALL_TIME_ZONES_URI, 436),
+    @ddt.data((ALL_TIME_ZONES_URI, 439),
               (COUNTRY_TIME_ZONES_URI, 28))
```

Comment:
A comment here about when this number might have to change?

Reply:
Sure, adding.